cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/store/remote/RemoteLineageWriterTest.java

```diff
@@ -14,14 +14,13 @@
  * the License.
  */
 
-package co.cask.cdap.internal.app.store;
+package co.cask.cdap.internal.app.store.remote;
```

Comment:
I know this is not your PR, but is everything in ``...app.store.remote``? Not all of this is "Store" stuff right? Should we just move everything to co.cask.cdap.internal.remote (not sure if you want to indicate secure/impersonation there too)?